core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -105,7 +105,9 @@ export class ResultPanelComponent {
       if (event.current.state === ExecutionState.Completed || event.current.state === ExecutionState.Running) {
         const sinkOperators = this.workflowActionService.getTexeraGraph().getAllOperators()
           .filter(op => op.operatorType.toLowerCase().includes('sink'));
-        if (sinkOperators.length > 0) {
+        if (sinkOperators.length > 0 && ! this.resultPanelOperatorID) {
```

Comment:
as I read here, this came into my mind:

what happens if I disable a view result operator after the workflow is successfully executed, and I want to see the results? Can I still view results?

Reply:
I believe you can